shared/src/commonMain/kotlin/co/nimblehq/mark/kmmic/data/service/auth/model/AuthLoginParams.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.mark.kmmic.data.service.auth.model
+
+import co.nimblehq.mark.kmmic.BuildKonfig
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+private const val GRANT_TYPE_PASSWORD = "password"
+
+@Serializable
+internal data class AuthLoginParams(
+    @SerialName("grant_type")
+    val grantType: String = GRANT_TYPE_PASSWORD,
+    val email: String,
```

Comment:
It's prefered to put `@SerialName` to all properties

Reply:
@suho updated at [b5c86d3](https://github.com/markgravity/kmm-ic/pull/67/commits/b5c86d34cf43b9e1cac1bdf1dc2ae0f36ddf159a)